### ISSUE TITLE
Create SidebarLeftScrollbar component

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#662]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

As seen in #660, the behavior of the scrollbar for the left sidebar is extremely problematic and we're not going to be able to get close to a good UI without building the scrollbar ourselves (not ideal, but is the situation). The styling should roughly follow [the custom CSS class](https://github.com/activist-org/activist/blob/6699a1ec97aa1faa1d43c246832ef2166e6369fb/frontend/assets/css/tailwind.css#L13), with the conditions for the sidebar content being scrollable then triggering its appearance in [SidebarLeft](https://github.com/activist-org/activist/blob/main/frontend/components/sidebar/left/SidebarLeft.vue) ([see conditions](https://github.com/activist-org/activist/blob/6699a1ec97aa1faa1d43c246832ef2166e6369fb/frontend/components/sidebar/left/SidebarLeft.vue#L298)).

### Contribution

This is a really important issue. Would be happy for the support, but please understand that we need this to be done very soon. We can also work on this at Code Night 🌙 on Saturday 😊

**Comment by andrewtavis** [comment]
917712f sent along the base styling for the scrollbar and positioned it appropriately to the left sidebar. We now need the height to be set dynamically given the scroll ratio, and then it needs to be moved given a scroll check :) @UnknownSean8, would you want to take it from here?

**Comment by andrewtavis** [comment]
CC @rigojr :)

**Comment by andrewtavis** [comment]
Closing this given 9646fa0 and 1f7d931. I've updated the designs so that there's now a gap in scrollbar footer if it's scrollable :) This allows us to use just the normal scrollbar and not have to do a special component for this.

What specifically is needed now is for the margin in the sidebar layout to increase if the content is scrollable, as as of now the width increases and we don't lose the margin completely, but it's smaller than expected.

Thanks for the strong words against this, @rigojr! Nice to be held back from a worm hole 🪱
